packages/react/src/primitives/Breadcrumbs/BreadcrumbLink.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react';
+import classNames from 'classnames';
+
+import { classNameModifierByFlag } from '../shared/utils';
+import {
+  BreadcrumbsLinkProps,
+  ForwardRefPrimitive,
+  Primitive,
+  BaseBreadcrumbLinkProps,
+} from '../types';
+import { ComponentClassNames } from '../shared/constants';
+import { Link } from '../Link';
+import { BreadcrumbsContext } from './BreadcrumbsContext';
+
+const BreadcrumbLinkPrimitive: Primitive<BreadcrumbsLinkProps, 'span'> = (
+  { className, children, href, ...rest },
+  ref
+) => {
+  const { isCurrent, isDisabled } = React.useContext(BreadcrumbsContext);
+  const componentClasses = classNames(
+    ComponentClassNames.BreadcrumbsLink,
+    classNameModifierByFlag(
+      ComponentClassNames.BreadcrumbsLink,
+      'current',
+      isCurrent
+    ),
+    classNameModifierByFlag(
+      ComponentClassNames.BreadcrumbsLink,
+      'disabled',
+      isDisabled
+    ),
+    className
+  );
+
+  const linkProps = {
+    'aria-current': isCurrent ? 'page' : undefined,
+    'aria-disabled': isDisabled ? 'true' : undefined,
```

Comment:
I left a previous comment about this before the change to output the current element as text, but do we have a use case intended for passing `aria-disabled` to the link? If not, I'd remove it all together. If we need it for something weird we _might consider_ this: https://www.scottohara.me/blog/2021/05/28/disabled-links.html